frontend/src/component/ui/Spinner.tsx

```diff
@@ -1,19 +1,19 @@
 import React, { Component } from 'react';
 
-import './Spinner.scss';
+import styles from './Spinner.module.scss';
 
 export class Spinner extends Component {
   render() {
     return (
-      <div className={'spinner'}>
-        <div className={'solar-system'}>
-          <div className={'earth-orbit orbit'}>
-            <div className={'planet earth'} />
-            <div className={'venus-orbit orbit'}>
-              <div className={'planet venus'} />
-              <div className={'mercury-orbit orbit'}>
-                <div className={'planet mercury'} />
-                <div className={'sun'} />
+      <div className={styles['spinner']}>
+        <div className={styles['solar-system']}>
```

Comment:
```suggestion
        <div className={styles.solarSystem}>
```
Does this work?

Reply:
No it does not. I would have to rename the class to `solarSystem` in the SCSS module.

Reply:
@Coteh Okey. I will let the PR go through this time. We will have to figure out a styling guide in the future.